Mathlib/Algebra/Module/Equiv.lean

```diff
@@ -334,7 +335,7 @@ set_option quotPrecheck false in
 /-- The notation `e₁ ≪≫ₗ e₂` denotes the composition of the linear equivalences `e₁` and `e₂`. -/
 infixl:80 " ≪≫ₗ " =>
   @LinearEquiv.trans _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ (RingHom.id _) (RingHom.id _) (RingHom.id _)
-    (RingHom.id _) (RingHom.id _) (RingHom.id _) RingHomCompTriple.ids RingHomCompTriple.ids
+    (RingHom.id _) (RingHom.id _) (RingHom.id _) RingHomCompTriple.ids
```

Comment:
I'm a bit worried by this; why does Lean3 take an extra instance argument here?

Reply:
Should we give `LinearEquiv.trans` an extra instance?

Reply:
Yes, I think we should; though I can't work out a nice way to do so without repeating all the `variables` lines

Reply:
I created [a Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/forcing.20inclusion.20of.20unused.20arguments/near/328480100)